sim/verilator-uhdm/meta.yaml

```diff
@@ -1,13 +1,13 @@
 # Use `conda-build-prepare` before building for a better version string.
-{% set version = '%s_%04i_%s'|format(GIT_DESCRIBE_TAG|replace('v','') or '0.X.X', GIT_DESCRIBE_NUMBER|int,   GIT_DESCRIBE_HASH or 'gUNKNOWN') %}
+{% set version = GIT_DESCRIBE_HASH %}
```

Comment:
Don't change this line. It doesn't influence the final version of the package as it is set by conda-build-prepare in CI.